modules/installation-user-infra-machines-advanced-customizing-live-ca-certs.adoc

```diff
@@ -32,6 +32,12 @@ $ coreos-installer pxe customize rhcos-<version>-live-initramfs.x86_64.img \
 ----
 endif::[]
 +
+[IMPORTANT]
+====
+The `coreos.inst.ignition_url` kernel parameter does not work with the `--ignition-ca` flag.
+Use of the `--dest-ignition` flag is required to create a customized image for each cluster.
```

Comment:
```suggestion
Use the `--dest-ignition` flag to create a customized image for each cluster.
```

Just a possibly more direct suggestion for phrasing.

Reply:
While I like the more direct wording, I feel that it's important to state that this flag is explicitly required. Direct wording implies this, but does not state it.

Reply:
```suggestion
You must use of the `--dest-ignition` flag is required to create a customized image for each cluster.
```